contacts24/db.py

```diff
@@ -50,40 +51,46 @@ def save_address_book(address_book: AddressBook, filename: str = ADDRESSBOOK_FIL
 
 #endregion
 
-#region Record Serialization Helper
+#region Notes
 
-def record_serialization(record: Record) -> dict:
-    """Record to dictionary
+def get_notes(filename: str = NOTES_FILE) -> Notes:
+    """Get notes from fixed json file (from config.NOTES_FILE)
 
-    Args:
-        record (Record): input
+    Raises:
+        AppError: All errors wrapped in AppError
 
     Returns:
-        dict: dictionary based on input record
+        Notes: List of notes
     """
-    return {
-        "name": record.name.value,
-        "phones": [phone.value for phone in record.phones],
-        "birthday": record.birthday.value if record.birthday else None,
-    }
+    
+    notes = Notes()
+    
+    try:
+        with open(filename, "r") as file:
+            notes_list = json.load(file)
+            for note_dict in notes_list:
+                note = note_deserialization(note_dict)
+                notes.add_note(note)
+    except Exception as e:
+        raise AppError(e)
+    
+    return notes
 
-def record_deserialization(data: dict) -> Record:
-    """Deserialization Record from json
+def save_notes(notes: Notes, filename: str = NOTES_FILE) -> None:
+    """Save notes to json file (from config.NOTES_FILE)
 
     Args:
-        data (dict): Record in json
+        notes (Notes): Notes to save
 
-    Returns:
-        Record: instance of class Record
+    Raises:
+        AppError: Any errors (missing file, incorrect json) wrapped in AppError
     """
-    record = Record(data["name"])
-    
-    for phone in data["phones"]:
-        record.add_phone(phone)
-    
-    if data["birthday"]:
-        record.add_birthday(data["birthday"])
-    
-    return record
 
-#endregion
+    try:
+        with open(filename, "w") as file:
+            notes_list = [note_serialization(note) for note in notes.data.values()]
+            json.dump(notes_list, file)
+    except Exception as e:
+        raise AppError(e)
+
+#endregion
```

Comment:
👮‍♂️ newline police 👮‍♂️

Reply:
@IhorNi  ?
didn't get it

Reply:
Shows no newline at the end of the file, but if checks passed, that's probably okay